requirements/requirements.txt

```diff
@@ -1,6 +1,7 @@
-ambition-django-timezone-field>=2.0.4
-fleming>=0.6.0
-django-manager-utils>=2.0.0
+#ambition-django-timezone-field>=2.0.4
```

Comment:
yeah, i would like to switch back to the main repo instead of our fork if possible. i think @geophphrie looked into our fork recently to see what was different about it, and i am hoping we don't have to maintain it anymore. @geophphrie do you have any input here?